cmd/create-index-snapshot/README.md

```diff
@@ -10,8 +10,18 @@ This output can be used to restore a database from a previous snapshot.
 
 ```sh
 Usage of ./create-index-snapshot:
-  -d, --dir string         path to badger database
-  -l, --log-level string   log level for JSON logger (default "info")
+  -d, --dir string   path to badger database (required)
+  -h, --hex          use hex output
+  -l, --log string   log level for JSON logger (default: "info")
+  -o, --out string   output file to write to (overwrites existing)
+```
+
+## Examples
+
+```console
+$ ./create-index-snapshot -d /path/to/index -o file.bin       # write snapshot to file (binary)
+$ ./create-index-snapshot -d /path/to/index -o file.txt --hex # write snapshot to file (hex)
+$ ./create-index-snapshot -d /path/to/index                   # writes snapshot to stdout (hex)
 ```
```

Comment:
This is quite convoluted IMO. No magical idea on how to improve it for now and it won't be used super often so I guess it's OK as it is 🤔

Reply:
I don't like it much either. One way to have it simpler is for it to be `hex` by default, then the only flag that changes that behaviour is to have it binary. E.g.

```console
$ ./create-index-snapshot -d /path/to/index                     # write snapshot to stdout (hex)
$ ./create-index-snapshot -d /path/to/index -o file.txt         # write snapshot to file (hex)
$ ./create-index-snapshot -d /path/to/index -o file.bin --raw   # write snapshot to file (binary)
```

It's simpler but kind of the opposite from what Max wanted :D